implementations/micrometer-registry-statsd/src/test/java/io/micrometer/statsd/internal/BufferingFluxTest.java

```diff
@@ -66,7 +66,7 @@ void bufferMultipleStrings() {
                 "fourteen bytes"
         );
 
-        Flux<String> buffered = BufferingFlux.create(source, "\n", 27, Long.MAX_VALUE);
+        Flux<String> buffered = BufferingFlux.create(source, "\n", 27, 1000);
```

Comment:
Passing `Long.MAX_VALUE` here would cause an overflow because `Long.MAX_VALUE` milliseconds eventually tries to get converted to nanoseconds.